### DISCUSSION TITLE
Stack icons vertically, even removing icons

### DISCUSSION THREAD

**Discussion by cavalier3154** [valinet/ExplorerPatcher#2734] [Ideas]
First, let me thank you for the ExplorerPatcher! I don't know what Microsoft is thinking...
Second, I didn't find this problem laid out in the way I wanted exactly, so if this topic was discussed please point me to the conclusion and apologies.
- Problem: When multiple windows are combined and I right click the combined icon, the different tasks open in a HORIZONTAL list like this, obstructing/truncating the end of the path:
![Screenshot 2024-01-21 220105](https://github.com/valinet/ExplorerPatcher/assets/156568207/bc0c25e1-bfb7-46c6-952c-eb08ccf3594a)
- Desired behavior: I do want to see the full path because I have the same folder name in different locations. This used to work well in windows 7 and I believe 10, where the list is VERTICAL and the full path is shown (the icon expands to show the full path).
- Known Workaround: I have seen very few people complain about it, the the only solution is to hover over each one of those icons to reveal the full path. That's slow so not practical.
Can you get that behavior back???
Thanks!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by cavalier3154** [comment]
Thanks for taking a look. Here are the answers:
-- Windows:
Edition	Windows 11 Home
Version	23H2
OS build	22631.3007
Experience	Windows Feature Experience Pack 1000.22681.1000.0

-- I have tried to change back and forth a number of explorer patcher properties to see if anything helps. I pasted the current settings at the bottom.  I tried both Windows 10 and windows 11 for the taskbar. Both give the same result.

-- I uninstalled both Open Shell and Explorer Patcher, and reinstalled the latest stable patcher: 22621.2861.62.2 before filing my report and taking the screenshot earlier. 
(Later, I reinstalled latest open shell 4.4.191 - but it has no effect on the taskbar).
I don't remember if I had done other registry changes - sorry- it has been a while. 
I don't think I have any other software that changes the UI. 

Having said that, this issue has irritated me for a while and I have been trying to find a fix. 

Is this not what you see on your windows machine?! 

[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ImportOK"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OldTaskbar"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_TaskbarPosition"=dword:00000003
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_MMTaskbarPosition"=dword:00000003
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Search]
"SearchboxTaskbarMode"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"ShowTaskViewButton"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_AutoHideTaskbar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OrbStyle"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OldTaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"MMOldTaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"TaskbarGlomLevel"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"MMTaskbarGlomLevel"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarSmallIcons"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"SkinMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"CenterMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"FlyoutMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\TabletTip\1.7]
"TipbandDesiredVisibility"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"ShowSecondsInSystemClock"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"HideControlCenterButton"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarSD"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ReplaceNetwork"=dword:00000002
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Control Panel\Settings\Network]
"ReplaceVan"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows NT\CurrentVersion\MTCUVC]
"EnableMtcUvc"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\ImmersiveShell]
"UseWin32TrayClockExperience"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\ImmersiveShell]
"UseWin32BatteryFlyout"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"IMEStyle"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_RegisterAsShellExtension"=dword:00000001
[HKEY_CURRENT_USER\Software\Classes\CLSID\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\InprocServer32]
@=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"LegacyFileTransferDialog"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UseClassicDriveGrouping"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_FileExplorerCommandUI"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableImmersiveContextMenu"=dword:00000000
[-HKEY_CURRENT_USER\Software\Classes\CLSID\{056440FD-8568-48e7-A632-72157243B55B}\InprocServer32]
@=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableModernSearchBar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ShrinkExplorerAddressBar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"HideExplorerSearchBar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"MicaEffectOnTitlebar"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"Start_ShowClassicMode"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_Start_MaximumFrequentApps"=dword:00000006
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StartPage]
"MonitorOverride"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartUI_ShowMoreTiles"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartUI_EnableRoundedCorners"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ForceStartSize"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_NoStartMenuMorePrograms"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer]
"AltTabSettings"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_PeopleBand"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"LastSectionInProperties"=dword:0000000a
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ClockFlyoutOnWinC"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ToolbarSeparators"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"PropertiesInWinX"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"NoMenuAccelerator"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableOfficeHotkeys"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableWinFHotkey"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableRoundedCorners"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableAeroSnapQuadrants"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"Start_PowerButtonAction"=dword:00000002
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectSystemToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectProgramsAndFeaturesToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectDateAndTimeToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectNotificationIconsToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdatePolicy"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdatePreferStaging"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateAllowDowngrades"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateURL"=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateURLStaging"=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"AllocConsole"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"Memcheck"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"TaskbarAutohideOnDoubleClick"=dword:00000000
[HKEY_CURRENT_USER\Control Panel\Desktop]
"PaintDesktopVersion"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ClassicThemeMitigations"=dword:00000000
[-HKEY_CURRENT_USER\Software\Classes\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}\InprocServer32]
@=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"NoPropertiesInContextMenu"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"EnableSymbolDownload"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"PinnedItemsActAsQuickLaunch"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"RemoveExtraGapAroundPinnedItems"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ExplorerReadyDelay"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
"XamlSounds"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"Language"=dword:00000000

**Comment by cavalier3154** [comment]
In my first post, I only showed the titles of the icons. I do see the full icon and their sizes limits the width. That's what I meant by removing the icon, here is a bit more of the view. Sorry if this was not clear in my first post. If we get rid of the icons and stack them vertically and expand the width to cover all the path, that would be great! (That's what was possible earlier).

![Screenshot 2024-01-22 002734](https://github.com/valinet/ExplorerPatcher/assets/156568207/c4fc93b0-67c0-4c9a-a47d-3cb8bfaf3e39)

**Comment by pyrates999** [comment]
I can't tell what you changed by the registry.  Please post screen shots of the EP Properties application settings.  There should be 10 screen shots.

**Reply by cavalier3154** [reply]
![Screenshot 2024-01-22 003511](https://github.com/valinet/ExplorerPatcher/assets/156568207/a5811856-9829-4cd0-b00b-7546007fda9c)
![Screenshot 2024-01-22 003453](https://github.com/valinet/ExplorerPatcher/assets/156568207/42f29141-dbe4-4b55-837a-094988295e89)
![Screenshot 2024-01-22 003448](https://github.com/valinet/ExplorerPatcher/assets/156568207/abb1fa6f-4aa8-430a-9a6b-520bd76803ff)
![Screenshot 2024-01-22 003443](https://github.com/valinet/ExplorerPatcher/assets/156568207/d583b81f-6a89-4708-b3cf-73ae00a1b0cf)
![Screenshot 2024-01-22 003437](https://github.com/valinet/ExplorerPatcher/assets/156568207/2a294f8a-bc7e-4d7c-ba77-707cd0cfe678)
![Screenshot 2024-01-22 003428](https://github.com/valinet/ExplorerPatcher/assets/156568207/3a22cfd2-57bf-4dc7-8d41-1704b8620eac)
![Screenshot 2024-01-22 003423](https://github.com/valinet/ExplorerPatcher/assets/156568207/ddfdb9c6-0d27-406d-b4bb-08d32812f417)
![Screenshot 2024-01-22 003417](https://github.com/valinet/ExplorerPatcher/assets/156568207/dce2e0c8-838b-4611-90cb-c153e48891c9)
![Screenshot 2024-01-22 003411](https://github.com/valinet/ExplorerPatcher/assets/156568207/53457ce6-d504-4ace-91a5-42508254fc7c)
![Screenshot 2024-01-22 003401](https://github.com/valinet/ExplorerPatcher/assets/156568207/78e59e75-b055-4f02-a3ec-3c7ff272b099)
![Screenshot 2024-01-22 003341](https://github.com/valinet/ExplorerPatcher/assets/156568207/79e558e6-4477-4f5b-a9be-11a837ce61f6)

**Comment by pyrates999** [comment]
Untick **register as shell extension**.  You shouldn't be enabling that unless you know what you're doing: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Reply by cavalier3154** [reply]
Done.  
There is no relationship/effect on this issue though.

**Comment by pyrates999** [comment]
The only time when taskbar icons are stacked vertically is if you open a lot more instances of an icon, at least 10.  Windows always shows it horizontally if it can because it wants to show you a preview of it if it can.

**Comment by cavalier3154** [comment]
Thank you! I forgot it was a windows feature that kicks in after a certain number of windows open. I had changed the threshold in windows 10 a long time ago. 
In windows 11, the default threshold seems to be 16 now!  
I found instructions to set a lower threshold for windows 11. (I like 1, so that it disables the icons preview. )
We can close this thread.
Thanks again!

PS: IF anybody is interesting, this is one place that describes the registry change for windows 11:
https://www.elevenforum.com/t/restart-windows-explorer-exe-process-in-windows-11.665/
